contest/b_nom_de_groupe/sol/b_nom_de_groupe.cpp

```diff
@@ -0,0 +1,49 @@
+// Solution by Louis Sugy
+
+#include <iostream>
+#include <string>
+
+int score(std::string name) {
```

Comment:
Not sure it's a big overhead, but I think that you don't need to copy here (`const std::string&`).

Reply:
Oh, how did I miss this? 😞 
It's not much but it's a useless cost